openc3/python/openc3/utilities/logger.py

```diff
@@ -25,14 +25,51 @@
 
 class LoggerMeta(type):
     def __getattribute__(cls, func):
-        if func == "instance" or func == "instance_mutex" or func == "my_instance":
+        if (
+            func == "instance"
+            or func == "instance_mutex"
+            or func == "my_instance"
+            or func == "scope"
+            or func == "__dict__"
+        ):
```

Comment:
Would like to see this as an array of values with a comment. Then you can do if func in ATTR_NAMES: or whatever you call it. Comment on where these names come from.

Same for the next three examples.